### ISSUE TITLE
Consistency in exported TSV 3 formats  

### ISSUE THREAD

**Issue by hterhors** [webanno/webanno#329]
I wonder why the TSV-3 format has some inconsistencies. 

E.g. If there is an annotation over more than one token, then all annotations are marked with an ID.

```
13-23   675-686 projections Class1[8]   _   _
13-24   687-691 from    Class1[8]   _   _
13-25   692-695 the Class1[8]   _   _
```

However, if the annotation is just on one single token 
`13-29  603-617 chondroitinase  Class2  _   _`
the ID is missing. But is somehow increased in the background?! 

This makes it hard to parse, since you need to count the annotations yourself.
Same goes for the reference-pointer in the last column. Sometimes the IDs are explicitly written, sometimes not. 
`13-17  632-638 enzyme  Class3  describes|uses  13-3|13-23[8_0]`

Further it is a bit counterintuitive that the ID is set to '0' if it points to itself.
I think it would be much nicer, and easier to understand if all IDs are always explicitly written.

Best regards,
Hendrik

**Comment by reckart** [comment]
@seyyaw I think we had discussed that some time ago and I believe I was also in favor of having the IDs there consistently. If I remember correctly, you had argued that you only wanted to have the IDs when they are necessary for disambiguation, right?

**Comment by seyyaw** [comment]
Yes.We came the idea of having an ID for disambiguation. we do not need an ID when it is not ambigous mainly due readability. Since the ID is on the same column as the annotation, it is easy to parse (a matter of an ID exists or not). But, if people are interested to have an ID all the time, we can include checkbox to select specifically. For post Webanno 3.0

**Comment by reckart** [comment]
I'd rather not have a checkbox because that would require that we introduce import/export configurations for each format in the UI. I'd rather have the ID always and now for 3.0.0.

**Comment by reckart** [comment]
I have added more documentation on the disambiguation IDs in general and also in the context of relations:

* https://github.com/webanno/webanno/blob/master/webanno-doc/src/main/asciidoc/user-guide/webannotsv.adoc#disambiguation-ids
* https://github.com/webanno/webanno/blob/master/webanno-doc/src/main/asciidoc/user-guide/webannotsv.adoc#relation-annotations

I think that should cover it for the moment.

Actually, instead of having more disambiguation IDs, I think it would actually be better to have less. For future TSV formats, it would be nice if disambiguation IDs were used only to track multi-unit annotations. For cross-unit references, it should IMHO be sufficient to come up with an ID system that uniquely identifies the column coordinates (i.e. unit ID, column number, stacking position). That information can all be included in a reference, avoiding the need for a disambiguation ID in such cases.
